index.rst

```diff
@@ -178,6 +234,8 @@ If these types of tasks are performed from the LOVE interface, then the same res
 Discovering Available Override Configurations
 ---------------------------------------------
 
+It is also possible to override the defaults using one of the available overrides.
+However, first one must determine which overrides are available to the CSC.
```

Comment:
This seems a tad repetitive. Consider replacing "However, first one must..." with "One way to determine which overrides are available is using a Jupyter notebook server".

I am surprised that is considered easier than querying the EFD via the Chronograf interface or (less authoritatively) the github repo for the config.

I'm glad a LOVE interface is planned. Surely that'll be the first choice for operators.

Reply:
I think part of the issue is that it's hard to explain how to use the chronograf interface as a whole, and it's beyond the scope of this document.
I re-arranged this section a bit and took the references to Chronograf from the bottom of this subsection to the top.